Specs/DateTime/English/DateParser.json

```diff
@@ -1634,9 +1563,8 @@
 {
   "Input": "I went back Thursday the 21st",
   "Context": {
-    "ReferenceDateTime": "2017-09-27T17:25:49.8080661+08:00"
+    "ReferenceDateTime": "2017-09-27T00:00:00"
```

Comment:
Does keeping the original value cause the spec test to fail? Or was the change just for consistency/cleanup?

Reply:
Just a little cleanup, these tests only use the `date` information